sympy/physics/mechanics/kane.py

```diff
@@ -843,31 +891,40 @@ def kanes_equations(self, bodies, loads=None):
                     'kanes_equations(loads, bodies) > kanes_equations(bodies, loads).',
                     issue=10945, deprecated_since_version="1.1").warn()
 
+        # TODO : Why is it optional to pass in the kinematic differential
+        # equations? I'm not sure what the use case is without them.
```

Comment:
As mentioned in the first docstring the kinematic differential equations can be left out and it would be assumed that u = q' (assuming the code actually treats this). If that is the case, however, I would think that the formation of the equations of motion should still work.